gnucash/import-export/import-main-matcher.c

```diff
@@ -497,6 +511,20 @@ gnc_gen_trans_init_view (GNCImportMainMatcher *info,
                      G_CALLBACK(gnc_gen_trans_row_changed_cb), info);
 }
 
+static gboolean
+is_color_light (GdkRGBA *color)
+{
+    gboolean is_light = FALSE;
+
+    // Counting the perceptive luminance - human eye favors green color...
+    double a = (0.299 * color->red + 0.587 * color->green + 0.114 * color->blue);
+
+    if (a > 0.5)
+        is_light = TRUE;
+
+    return is_light;
+}
```

Comment:
Hmm, writing the same function twice is never a good idea. Can you move it to somewhere both locations can use it ? First suggestion would be gnucash/gnome-utils/gnc-gtk-utils.[ch]

Reply:
Yes, just did not want to change another file if this approach was wrong.